Shared/Samples/Set map image layer sublayer visibility/SetMapImageLayerSublayerVisibilityView.swift

```diff
@@ -0,0 +1,104 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct SetMapImageLayerSublayerVisibilityView: View {
+    /// The tracking status for the loading operation.
+    @State private var isLoading = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    @State private var map: Map = {
+        let map = Map()
+        return map
+    }()
+    
+    @State private var mapImageLayer: ArcGISMapImageLayer = {
+        let imageLayer = ArcGISMapImageLayer(url: .arcGISMapImageLayerSample)
+        return imageLayer
+    }()
+    
+    @State private var sublayerOptions: [SublayerOption] = []
+    
+    var body: some View {
+        MapViewReader { proxy in
+            MapView(map: map)
+                .task {
+                    // Adds the map image layer to the map.
+                    map.addOperationalLayer(mapImageLayer)
+                    do {
+                        // Loads the map image layer.
+                        try await mapImageLayer.load()
+                        sublayerOptions = mapImageLayer.mapImageSublayers.enumerated().map { index, mapImageSublayer in
+                            SublayerOption(
+                                name: mapImageSublayer.name,
+                                id: index,
+                                isEnabled: mapImageSublayer.isVisible,
+                                sublayer: mapImageSublayer
+                            )
+                        }
+                        await proxy.setViewpoint(Viewpoint(center: Point(x: -11e6, y: 6e6, spatialReference: .webMercator), scale: 9e7))
+                    } catch {
+                        self.error = error
+                    }
+                }
+                .toolbar {
+                    ToolbarItem(placement: .bottomBar) {
+                        Menu {
+                            ForEach($sublayerOptions, id: \.id) { $sublayerOption in
+                                Toggle(isOn: $sublayerOption.isEnabled) {
+                                    Text(sublayerOption.name)
+                                }
+                                .onChange(of: sublayerOption.isEnabled) {
+                                    sublayerOption.sublayer.isVisible = sublayerOption.isEnabled
+                                }
+                            }
+                        } label: {
+                            Text("Sublayers")
+                        }
+                    }
+                }
+                .errorAlert(presentingError: $error)
+        }
+        
+    }
+}
+
+private struct SublayerOption: Equatable, Identifiable {
+    var name: String
+    let id: Int
+    var isEnabled = true
```

Comment:
This should be `isVisible`
```suggestion
    var isVisible = true
```